editor/editor_undo_redo_manager.cpp

```diff
@@ -42,16 +42,19 @@
 EditorUndoRedoManager *EditorUndoRedoManager::singleton = nullptr;
 
 EditorUndoRedoManager::History &EditorUndoRedoManager::get_or_create_history(int p_idx) {
-	if (!history_map.has(p_idx)) {
+	HashMap<int, EditorUndoRedoManager::History>::Iterator I = history_map.find(p_idx);
+	if (!I) {
 		History history;
 		history.undo_redo = memnew(UndoRedo);
 		history.id = p_idx;
 		history_map[p_idx] = history;
 
 		EditorNode::get_singleton()->get_log()->register_undo_redo(history.undo_redo);
 		EditorDebuggerNode::get_singleton()->register_undo_redo(history.undo_redo);
+		return history;
```

Comment:
Fails with GCC, returning a reference to a local variable. Should likely drop the `&` from the signature?